devops/scripts/install_build_tools.sh

```diff
@@ -26,3 +26,11 @@ apt update && apt install -yqq \
       libhwloc-dev \
       libzstd-dev \
       time
+
+# To obtain latest release of spriv-tool.
+# Same as what's done in SPRIV-LLVM-TRANSLATOR:
+# https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/cec12d6cf46306d0a015e883d5adb5a8200df1c0/.github/workflows/check-out-of-tree-build.yml#L59
+. /etc/os-release
+curl -L "https://packages.lunarg.com/lunarg-signing-key-pub.asc" | apt-key add -
+echo "deb https://packages.lunarg.com/vulkan $VERSION_CODENAME main" | sudo tee -a /etc/apt/sources.list
```

Comment:
we can probably remove `sudo` after the `|` because all the other commands assume current user is root